docs/advanced-features/open-telemetry.md

```diff
@@ -0,0 +1,109 @@
+---
+description: Learn how instrument your Next.js app with OpenTelemetry.
+---
+
+> **Note**: This feature is experimental, you need to explicitly opt-in by providing `experimental.instrumentationHook = true;` in your `next.config.js`.
+
+# OpenTelemetry in Next.js
+
+Observability is crucial for understanding and optimizing the behavior and performance of your Next.js app.
+
+As applications become more complex, it becomes increasingly difficult to identify and diagnose issues that may arise. By leveraging observability tools, such as logging and metrics, developers can gain insights into their application's behavior and identify areas for optimization. With observability, developers can proactively address issues before they become major problems and provide a better user experience. Therefore, it is highly recommended to use observability in your Next.js applications to improve performance, optimize resources, and enhance user experience.
+
+We will use terms like _Span_, _Trace_ or _Exporter_ throughout this doc, all of which can be found in [the OpenTelemetry Observability Primer](https://opentelemetry.io/docs/concepts/observability-primer/).
+
+> **Note:** We currently support observability bindings only in serverless functions.
+> We don't provide any for `edge` or client side code.
+
+## Getting Started
+
+Next.js supports OpenTelemetry, which is platform agnostic. You can change your observability provider without changing your code.
```

Comment:
Next.js supports OpenTelemetry instrumentation out of the box, (expand on what that means)

Reply:
I tried expanding it, but I'm unsure if that is enough. I don't want to turn it into a blogpost :)